themes/sensei-course-theme/single-quiz.php

```diff
@@ -11,11 +11,55 @@
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
+
 ?>
-<!-- wp:sensei-lms/container {"className":"sensei-course-theme__quiz"} -->
-<div class="wp-block-sensei-lms-container sensei-course-theme__header sensei-course-theme__frame">
-	<!-- wp:sensei-lms/quiz-back-to-lesson /-->
-	<!-- wp:post-title /-->
-	<!-- wp:post-content /-->
+<!-- wp:sensei-lms/container {"className":"quiz"} -->
+<div class="quiz quiz-blocks">
+	<!-- wp:sensei-lms/container {"className":"sensei-course-theme__quiz", "tagName": "form"} -->
+	<form method="post" enctype="multipart/form-data" class="wp-block-sensei-lms-container sensei-course-theme__quiz">
```

Comment:
We'll have to implement this properly or look for a different approach if we enable editing this template in FSE.